src/main/kotlin/no/nav/familie/ks/sak/kjerne/behandling/steg/vilkårsvurdering/VilkårsvurderingSteg.kt

```diff
@@ -175,6 +178,20 @@ class VilkårsvurderingSteg(
         }
     }
 
+    private fun validerAtDetIkkeFinnesMerEnn2EndringerISammeMånedIBarnehageplassVilkår(vilkårsvurdering: Vilkårsvurdering) {
+        vilkårsvurdering.personResultater.filter { !it.erSøkersResultater() }.forEach { personResultat ->
+            if (personResultat.tilPeriodeResultater().any { periodeResultat ->
+                periodeResultat.vilkårResultater.count { it.vilkårType == Vilkår.BARNEHAGEPLASS } > 2
+            }
+            ) {
+                throw FunksjonellFeil(
+                    "Du har lagt inn flere  enn 2 endringer i barnehagevilkåret i samme måned. " +
```

Comment:
Ikke meningen å være kjempe pirkete, men det er dobbelmellom rom her 🙈 
```suggestion
                    "Du har lagt inn flere enn 2 endringer i barnehagevilkåret i samme måned. " +
```